### PR TITLE
Add Proposed address curfew for a RO user

### PR DESCRIPTION
**Pull request by romidane**
This PR only add a page template for now called `/hdc/proposedAddress/curfewAddressConfirmation/A0001XX`

Resolves [155](https://dsdmoj.atlassian.net/browse/LIC-155)

### CODE REVIEW

File: server/routes/config/proposedAddress.js [hunk 1]
```diff
@@ -45,6 +45,18 @@ module.exports = {
...
+
+    curfewAddressConfirmation: {
+        licenceSection: 'curfewAddress',
+        nextPath: null,
+        fields: [
+            {landLordHDCConsent: {}},
+            {hasElectricitySupply: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {homeVisitConducted: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {managedSafely: {}},
+            {managedSafelyReasons: {dependantOn: 'managedSafely', predicate: 'No'}}
```
Comment (whitfield-mj) [inline server/routes/config/proposedAddress.js]: Need to change all these to dependent 'e' not 'a'

File: server/views/proposedAddress/curfewAddressConfirmationForm.pug [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        form(method="post")
+            input(type="hidden" name="_csrf" value=csrfToken)
+            input(type="hidden" name="nomisId" value=nomisId || '')
+            div.pure-g.largePaddingBottom
+                div.pure-u-1.pure-u-sm-5-12.pure-g
+                    div.form-group.inline
+                        p Does the homeowner/landlord give informed consent to HDC at the address?
+
+                        div.multiple-choice(data-target="landLordHtcConsentForm")
+                            input#hdcConsentYes(type="radio" checked=data && data.landLordConsent === 'Yes' name="landLordHDCConsent" value="Yes")
```
Comment (whitfield-mj) [inline server/views/proposedAddress/curfewAddressConfirmationForm.pug]: There should always be data as default is {}

Comment (whitfield-mj) [reply server/views/proposedAddress/curfewAddressConfirmationForm.pug]: throughout this page

File: server/routes/config/proposedAddress.js [hunk 3]
```diff
@@ -45,6 +45,18 @@ module.exports = {
...
+        ]
+    },
```
Comment (whitfield-mj) [inline server/routes/config/proposedAddress.js]: I think this is in the wrong place. We're doing a route per 'journey' so I would have thought we would layout our config files in the same way

Comment (whitfield-mj) [reply server/routes/config/proposedAddress.js]: think it needs to be in config/licenceConditions.js

File: server/routes/config/proposedAddress.js (hunk 1 above)
Comment (romidane) [reply server/routes/config/proposedAddress.js]: Can't spell to save a kitten 😅

File: server/views/proposedAddress/curfewAddressConfirmationForm.pug (hunk 2 above)
Comment (romidane) [reply server/views/proposedAddress/curfewAddressConfirmationForm.pug]: fair point

Comment (atodd-noms) [reply server/views/proposedAddress/curfewAddressConfirmationForm.pug]: - curfewAddressForm
- confirmAddressForm
- confirmationForm
- curfewAddressConfirmationForm
- submitForm

I'm lost...

**Pull request merged**